gems/pending/VixDiskLib/VixDiskLibServer.rb

```diff
@@ -1,3 +1,9 @@
+#
+# This script doesn't run in the context of the Rails environment,
+# so the following load path manipulation is required.
+#
+$LOAD_PATH << File.expand_path(File.join(__dir__, ".."))
```

Comment:
This was implemented to get around the conflicts with the openssl libraries we had when VDDK was shipped with an outdated set (0.9.8).